src/api2/ComputeEngineEmpty.java

```diff
@@ -1,15 +1,24 @@
 package api2;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class ComputeEngineEmpty implements ComputeEngine {
-    private InputConfig input ;
-    public ComputeEngineEmpty (InputConfig input){
+    private input input ;
+    public ComputeEngineEmpty (input input){
         this.input = input;
     }
     @Override
-    public String compute(List<String> input) {
-        throw new UnsupportedOperationException("Unimplemented method 'compute'");
+    public List<Integer> compute(List<Integer> input) {
+        List<Integer> b = new ArrayList<Integer>();
+		//find the prime under the given
+		for(int i = 0; i < input.size(); i++) {
+			if ((input.get(i) % 2) != 0) {
```

Comment:
The logic isn't perfect, I pushed a change with a _potential_ fix, but as it stands we're not iterating through the list like we want to. This is gonna use the size of the array as the calculation rather than the actual individual values within it.